setup.py

```diff
@@ -70,11 +70,15 @@ def copy_file (self, filename, dirname):
       license="GPLv2+",
       scripts=['bin/sos', 'bin/sosreport'],
       data_files=[
-        ('share/man/man1', ['man/en/sosreport.1']),
+        ('share/man/man1', ['man/en/sosreport.1', 'man/en/sos-report.1',
+                            'man/en/sos.1', 'man/en/sos-collect.1',
+                            'man/en/sos-collector.1']),
         ('share/man/man5', ['man/en/sos.conf.5']),
         ],
-      packages=['sos', 'sos.report.plugins', 'sos.policies', 'sos.report'],
+      packages=['sos', 'sos.policies', 'sos.report', 'sos.report.plugins',
+                'sos.collector', 'sos.collector.clusters'],
       cmdclass={'build': BuildData, 'install_data': InstallData},
+      requires=['pexpect']
```

Comment:
This is required just for `sos collect` but not for other subcommand(s). Do we need this requirement for whole `sos`, then?

Reply:
My perspective here is that splitting the components out, or disabling them via not setting certain dependencies, is up to the downstream packagers. For us, I think it makes sense to have it default to required so that the entire functionality of the tool is available when you're running off of the current master.